pipeline-scripts/buildlib.groovy

```diff
@@ -1483,7 +1483,7 @@ def getChanges(yamlData) {
  *
  * }
  */
-def buildBuildingPlashet(version, release, el_major, include_embargoed, auto_signing_advisory=54765) {
+def buildBuildingPlashet(version, release, el_major, include_embargoed, auto_signing_advisory) {
```

Comment:
if there is no signing advisory in the group.yml `auto_signing_advisory` will be empty and signing won't occur. i'm thinking of 4.2 which still builds... we can either set it to `54765` if not given, or we can throw an error at this point (and update 4.2 `group.yml` to use `54765`). I think I'd prefer continuing to default if not specified.

Reply:
Right, let's continue to use the default signing advisory if not specified.